src/main/resources/assets/tinkersurvival/lang/zh_cn.json

```
@@ -11,107 +11,107 @@
+    "item.tinkersurvival.reinforced_jelled_slime_boots": "加固的黏液靴子",
     "item.tinkersurvival.tinkers_survival": "匠魂生存手册",
     "item.tinkersurvival.tinkers_survival_modpack": "匠魂生存整合包手册",
     "item.tinkersurvival.bandage": "绷带",
-    "item.tinkersurvival.cloth": "布片",
+    "item.tinkersurvival.cloth": "布条",
     "item.tinkersurvival.crude_bandage": "粗制绷带",
     "item.tinkersurvival.ointment": "软膏",
     "item.tinkersurvival.plant_paste": "植物浆",
     "item.tinkersurvival.wooden_cup": "神秘的木杯子",
-    "block.tinkersurvival.rock_stone_block": "小块石头",
-    "block.tinkersurvival.stone_loose_rock": "松散的石子堆 (石头)",
-    "block.tinkersurvival.andesite_loose_rock": "松散的石子堆 (安山岩)",
-    "block.tinkersurvival.diorite_loose_rock": "松散的石子堆 (闪长岩)",
-    "block.tinkersurvival.granite_loose_rock": "松散的石子堆 (花岗岩)",
-    "block.tinkersurvival.sandstone_loose_rock": "松散的石子堆 (砂岩)",
-    "block.tinkersurvival.red_sandstone_loose_rock": "松散的石子堆 (红砂岩)",
-    "jei.tinkersurvival.description.rock": "可以在地上找到这些石块，与小刀合成可以获得燧石碎片。",
-    "jei.tinkersurvival.description.plant_fiber": "植物纤维会在用小刀破坏草时掉落。它们可以制作用于工具的植物绳和用于绷带药膏的植物浆",
-    "jei.tinkersurvival.description.stick": "木棍可以在破坏树叶时掉落，用小刀可以增加获得木棍的几率。",
-    "jei.tinkersurvival.description.flint_shard": "燧石碎片可以通过在任意石质方块表面敲击燧石获得。用燧石击打任意石质方块，它也许会碎成两块燧石碎片。",
+    "block.tinkersurvival.rock_stone_block": "小块岩石",
+    "block.tinkersurvival.stone_loose_rock": "松散的岩石堆（石头）",
+    "block.tinkersurvival.andesite_loose_rock": "松散的岩石堆（安山岩）",
+    "block.tinkersurvival.diorite_loose_rock": "松散的岩石堆（闪长岩）",
+    "block.tinkersurvival.granite_loose_rock": "松散的岩石堆（花岗岩）",
+    "block.tinkersurvival.sandstone_loose_rock": "松散的岩石堆（砂岩）",
+    "block.tinkersurvival.red_sandstone_loose_rock": "松散的岩石堆（红砂岩）",
+    "jei.tinkersurvival.description.rock": "你可以在地上捡到这些岩石，用燧石小刀可以将它切割成燧石碎片",
+    "jei.tinkersurvival.description.plant_fiber": "用小刀斩断草时会掉落植物纤维。植物纤维可以合成用于制作工具的草绳和制作绷带软膏的植物浆。",
+    "jei.tinkersurvival.description.stick": "破坏树叶时会掉落木棍。使用任何小刀都可以增加木棍掉落的概率。",
+    "jei.tinkersurvival.description.flint_shard": "在任意岩石上敲击小块岩石会掉落燧石碎片。用小块岩石敲击任何岩石，都有一定概率碎成两个燧石碎片。",
     "tooltip.uselessTool1": "这个工具什么都挖不了！",
     "tooltip.uselessTool2": "只能用于合成。",
     "tooltip.uselessHoe1": "这把锄头什么都锄不动！",
     "tooltip.uselessWeapon1": "这把武器用起来像一根湿面条。",
     "tooltip.uselessBow1": "这把弓的弓弦被拉断了！",
-    "tooltip.uselessArmor1": "既定尺码：加加加大号。似乎不合身。",
+    "tooltip.uselessArmor1": "既定尺码：XXXL。似乎不合身。",
     "tooltip.cxp": "可用于合成",
     "message.notice": "注意",
     "message.warning": "警告",
-    "message.wrong_tool": "需要 %s 。",
-    "message.wrong_tool2": "嗯... %s ？",
-    "message.tool_broke": "你的 %s 坏了。",
+    "message.wrong_tool": "需要%s。",
+    "message.wrong_tool2": "嗯……%s？",
+    "message.tool_broke": "你的%s坏了。",
     "message.spawn_set": "一点都不困，只是设置了重生点！",
     "effect.tinkersurvival.stop_bleeding": "止血",
     "effect.tinkersurvival.zombie_essence": "僵尸精华",
-    "material.jelledslime": "冰冻史莱姆",
+    "material.jelledslime": "冻结黏液",
```

```suggestion
    "material.jelledslime": "凝结黏液",
```
maybe "congealed" is better than "frozen"?
may require a check on recipe or interop with other mods(possibly Tough as Nails) in-game

---

this material doesnt have any recipe and I even not see it in 1.18 TAN... so i used the translation of 1.12 version.